sandbox.sh

```diff
@@ -21,32 +18,39 @@ export PATH
 
 [ "$USE_NIX" ] && dune build @install
 
+# Use tezos-client to interact with the node.
+# See https://tezos.gitlab.io/introduction/howtouse.html#client
 tezos-client() {
   docker exec -t deku_flextesa tezos-client "$@"
 }
 
+# Necessary to write tezos smart contracts
 ligo() {
   docker run --rm -v "$PWD":"$PWD" -w "$PWD" ligolang/ligo:0.28.0 "$@"
 }
 
-if [ $mode  = "docker" ]
-then
+if [ $mode = "docker" ]; then
   RPC_NODE=http://flextesa:20000
 else
   RPC_NODE=http://localhost:20000
 fi
 
+# Tezos secret key?
 # This secret key never changes.
 SECRET_KEY="edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq"
 
+# Folder containing the data of each nodes (identity.json)
```

Comment:
It's more than just identity.json

Reply:
If it is more than `identity.json`, then what is the full meaning of it?

Reply:
@lykimq, I leave that as an exercise to the reader :wink:

Reply:
I see, it is more than `identity.json` because it contains others json files related to Deku node like: tezos.json, validators.json, etc.